packages/ui/src/SelectList.tsx

```diff
@@ -1,110 +0,0 @@
-// import React, {useContext} from "react";
```

Comment:
We can just remove this file if it's not needed

Reply:
hah yeah that's what i meant to do, my bad. done!